rpc/evm.go

```diff
@@ -112,7 +111,7 @@ func newTxArgs(params map[string]interface{}) ethapi.TransactionArgs {
 }
 
 // newEVM creates new instance of EVM with given parameters
-func (e *EvmExecutor) newEVM(msg eth.Message, hashErr *error) *vm.EVM {
+func (e *EvmExecutor) newEVM(msg core.Message, hashErr *error) *vm.EVM {
```

Comment:
Pass a pointer to `core.Message` here, you have to create it anyway on line `144`